### ISSUE TITLE
How did you decide on the epochs?

### ISSUE THREAD

**Issue by SamChuah** [JiaRenChang/PSMNet#171]
Hi author, 

Great work here. I re-implemented your method and I found out that after 10 epochs of training on Scene Flow dataset, the loss does not come to a plateau. Is there a reason why you start fine-tuning your model on KITTI before your training reach plateau with Scene Flow dataset?

Thank you

**Comment by JiaRenChang** [comment]
@SamChuah 
We indeed tried longer training.
However, it performed much worse.
It may be the much domain gap between sceneflow and KITTI.

**Comment by SamChuah** [comment]
@JiaRenChang Thanks for your reply. I have another question regarding finetuning and selecting the best model. In the paper you mentioned model pre-trained on Scene Flow was used to finetune on KITTI for 300 epochs and is evaluated. Then the training is prolonged to 1000 epochs and the best model is submitted to the KITTI benchmark. And in your "finetune.py" code, there is a section for validating the results of finetuning as your split the training set to 160/40. 

I am just wondering the results you showed in Table 2, 3 in your paper  was it the "Best" result or was it the final 300 epoch results? And did you prolong the training on the final 300 or the best results and do you do the training again or you adding in the other 40 images on top of the 160?
